src/container/interfaces.ts

```diff
@@ -81,6 +81,25 @@ export interface ContainerProps extends BaseComponentProps {
    * @visualrefresh `stacked` variant
    */
   variant?: 'default' | 'stacked';
+
+  /**
+   * Specifies an object of selectors and properties that are used to apply custom styles.
+   *
+   * - `root.background` (string) - (Optional) Background color of the container.
```

Comment:
I wonder if we want to word this (and similarly in other components too) "Background of the container", as that's technically how it's applied, not purely as a background color. Might make it clearer that you can also do things like gradients with this.